lib/_emerge/main.py

```diff
@@ -1304,6 +1350,27 @@ def emerge_main(args: Optional[list[str]] = None):
     emerge_config.action, emerge_config.opts, emerge_config.args = parse_opts(
         tmpcmdline
     )
+    if (
+        "--jobs-tmpdir-blocks-threshold" in emerge_config.opts
```

Comment:
I suppose we should check for merge-wait too?

Reply:
It's valid to use this option with or without merge-wait because either way the user could desire to  use disk pressure to limit starting of new jobs.